.github/workflows/tests.yml

```diff
@@ -76,8 +76,8 @@ jobs:
     - name: Install Python Deps
       if: steps.release.outputs.version == 0
       run: |
-        pip install --upgrade setuptools pip wheel
-        pip install -e .[test]
+        pip install --user --upgrade setuptools pip wheel
```

Comment:
This isn't necessary, as there is an implicit virtualenv.  If you're trying to fix a `EPERM` on Windows, then the correct fix is replacing `pip` with `python -m pip`

Reply:
changed